config/routes.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  get '/categories/:category_id/payment' => 'payments#index', as: :category_payments
+  get '/categories/:category_id/payment/new' => 'payments#new', as: :new_payment
+  post '/categories/:category_id/payment' => 'payments#create', as: :create_payment
+  devise_for :users
+  resources :payments
+  resources :categories
+  resources :users
```

Comment:
[REQUIRED] I noticed that you are using resources without `only` or `except` and this creates routes for actions that you are not using in your controllers like the `show` action in your users and categories controllers. Kindly ensure that resources only create routes that you have actions for in your controller with `only` or `except`.